lms/static/styles/frontend_apps.scss

```diff
@@ -31,6 +31,12 @@
 // Shared styles from frontend-shared package
 @use '@hypothesis/frontend-shared/styles';
 
+// This ensures that root components which set `width: 100%; height: 100%`
+// (eg. the `BasicLtiLaunchApp`) will fill the viewport.
+html, body {
+  height: 100%;
+}
```

Comment:
This rule used to be in `lms.css` but isn't needed for the static forms. It is not needed for all of the frontend mini-apps either and we might be able to implement a more narrowly scoped change for `BasicLtiLaunchApp` in future. For now I've just documented why it is needed.